crossing-repair.lic

```diff
@@ -102,8 +102,8 @@ class CrossingRepair
     ensure_copper_on_hand(@repair_withdrawal_amount, settings)
 
     # Merge, do not overwrite
-    @repair_info = { hometown_data['leather_repair'] => @equipment_manager.items.select(&:leather) }
-                   .merge(hometown_data['metal_repair'] => @equipment_manager.items.reject(&:leather)) { |_key, old, new| old + new }
+    @repair_info = { hometown_data['leather_repair'] => @equipment_manager.items.select(&:leather).reject(&:skip_repair) }
```

Comment:
I feel like at this point maybe EquipmentManager should have a `repairable_items(bool isLeather)` helper... but if it would only be used here, it's not a big deal.